src/app/legacy/containers/SocialEmbed/index.test.jsx

```diff
@@ -102,6 +102,34 @@ describe('SocialEmbedContainer', () => {
       unmount();
     });
 
+    it('should render a Twitter block and unmount correctly for STY page', () => {
```

Comment:
Was this a missing test? Not sure how this relates to this PR

Reply:
You're right actually. This folder is a little confusing as there is a CPS subdirectory specifically for CPS usage. The tests in there cover everything that we need, so I'll get rid of this one.